aiida_orca/parsers/__init__.py

```diff
@@ -24,25 +28,25 @@ def parse(self, **kwargs):
         If it would be an optimization run, the relaxed structure also will
         be stored under relaxed_structure key.
         """
-
         try:
             out_folder = self.retrieved
         except NotExistent:
             return self.exit_codes.ERROR_NO_RETRIEVED_FOLDER
 
-        fname_out = self.node.process_class._OUTPUT_FILE  #pylint: disable=protected-access
-        fname_relaxed = self.node.process_class._RELAX_COORDS_FILE  #pylint: disable=protected-access
-        # fname_traj = self.node.process_class._TRAJECTORY_FILE  #pylint: disable=protected-access
-        # fname_hessian = self.node.process_class._HESSIAN_FILE  #pylint: disable=protected-access
+        process_cls = self.node.process_class
+        fname_out = process_cls._OUTPUT_FILE  # pylint: disable=protected-access
+        fname_relaxed = process_cls._RELAX_COORDS_FILE  # pylint: disable=protected-access
 
-        if fname_out not in out_folder._repository.list_object_names():  #pylint: disable=protected-access
+        if fname_out not in out_folder.list_object_names():
             raise OutputParsingError('Orca output file not retrieved')
```

Comment:
I am wondering, why are we raising here, and not returning a non-zero exit code? (as done elsewhere in this file?)

Reply:
Fully agree, this was the original code. I will add an exit code for it

Reply:
Perfect, thank you!